view/css/main.css

```diff
@@ -83,10 +83,8 @@ header {
 
 #banner-listen {
     position: fixed;
-    text-align: center;
-    /* display: inline-block; */
-    top: 20%;
-    left: 56%;
+    top: 91.5%;
```

Comment:
you could better go with the property of bottom and right, rather than top and left since the mic icon is closer to the bottom-right corner. Please use this logic.